lib/dice_page.dart

```diff
@@ -1,101 +1,128 @@
+import 'package:dice_roller/src/bloc/i_dice_bloc.dart';
 import 'package:flutter/material.dart';
 import 'utils/assets.dart';
 import 'utils/constants.dart';
 
 class DicePage extends StatefulWidget {
+  final IDiceBloc bloc;
+
+  DicePage({
+    required this.bloc,
+  });
+
   @override
   _DicePageState createState() => _DicePageState();
 }
 
 class _DicePageState extends State<DicePage> {
-  late AssetImage topDice;
-  late AssetImage bottomDice;
-
-  @override
-  void initState() {
-    super.initState();
-    setState(() {
-      topDice = diceOne;
-      bottomDice = diceOne;
-    });
-  }
+  var title = "Try to match the dices";
```

Comment:
You can add this String into a Constant file like String file.

Reply:
Done

Reply:
done